sphinxarg/ext.py

```diff
@@ -93,7 +93,8 @@ def print_action_groups(data, nested_content, markdown_help=False, settings=None
     if 'action_groups' in data:
         for action_group in data['action_groups']:
             # Every action group is comprised of a section, holding a title, the description, and the option group (members)
-            section = nodes.section(ids=[action_group['title'].replace(' ', '-').lower()])
+            title_as_id = action_group['title'].replace(' ', '-').lower()
+            section = nodes.section(ids=[f"{id_prefix}-{title_as_id}"])
             section += nodes.title(action_group['title'], action_group['title'])
```

Comment:
I am observing a similar problem with labels: specifically with `autosectionlabel` and similar modules, e.g. `nbsphinx`.

This is specifically a problem because these modules generate these handles from rawsource or text, so to fix this a prefix also has to be added to `rawsource` and `text` in the title objects.

Reply:
Are you talking about the `nodes.title` object right here? Are `rawsource` and `text` the names of the arguments being passed to it?

Reply:
I don't really understand how we can add a prefix to `text` without changing the actual text of the title.

Reply:
Yes and I can go a step further https://github.com/sphinx-doc/sphinx/blob/3b907a1caab14fac7b1a66a0f3f30d969b0707b2/sphinx/ext/autosectionlabel.py#L41-L55 is where these links are created and at the end of that the warning gets generated.

But you are completely correct: changing the section label without changing the text is not possible right now.

Reply:
I appreciate you bringing it up either way. I think autosectionlabel may just not be compatible with this package. Does this PR and this option https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html#confval-autosectionlabel_prefix_document fix this at all?